$DragonFly: src/sbin/dhclient/client/Attic/dhclient.8.patch,v 1.1 2005/01/04 19:58:54 joerg Exp $
--- dhclient.8.orig	2004-06-10 19:59:12.000000000 +0200
+++ dhclient.8	2004-06-24 16:35:57.000000000 +0200
@@ -23,6 +23,10 @@
 .\"
 .\" $Id: dhclient.8,v 1.12.2.8 2004/06/10 17:59:12 dhankins Exp $
 .\"
+.\" Portions copyright (c) 2000 David E. O'Brien.
+.\" All rights reserved.
+.\" $FreeBSD: ports/net/isc-dhcp3-server/files/patch-client::dhclient.8,v 1.3 2004/03/22 23:33:09 des Exp $
+.\"
 .TH dhclient 8
 .SH NAME
 dhclient - Dynamic Host Configuration Protocol Client
@@ -33,12 +37,18 @@
 .I port
 ]
 [
+.B -D
+]
+[
 .B -d
 ]
 [
 .B -q
 ]
 [
+.B -v
+]
+[
 .B -1
 ]
 [
@@ -69,6 +79,10 @@
 relay
 ]
 [
+.B -i
+interval
+]
+[
 .B -n
 ]
 [
@@ -153,6 +167,15 @@
 configuration file or on the command line, and will ignore all other
 interfaces.
 .PP
+The
+.B -D
+flag causes
+.B dhclient
+to save the script it creates for use in conjunction with
+.B dhclient-script
+in
+.IR /tmp.
+.PP
 If the DHCP client should listen and transmit on a port other than the
 standard (port 68), the
 .B -p
@@ -176,6 +199,12 @@
 flag, followed by the IP address to send.   This is only useful for testing,
 and should not be expected to work in any consistent or useful way.
 .PP
+On FreeBSD, dhclient can be enabled to automatically handle the
+link status of the network card. Normally polling is done every
+five seconds. The polling interval can be set using the
+.B -i
+flag, followed by the numbers of seconds. Minimum is one second.
+.PP
 The DHCP client will normally run in the foreground until it has
 configured an interface, and then will revert to running in the
 background.   To run force dhclient to always run as a foreground
@@ -193,6 +222,10 @@
 .B -q
 flag prevents any messages other than errors from being printed to the
 standard error descriptor.
+.B -v
+flag turns on all messages.
+Opposite of
+.B -q .
 .PP
 The client normally doesn't release the current lease as it is not
 required by the DHCP protocol.  Some cable ISPs require their clients
